doc/manual/manual.md

```diff
@@ -688,9 +694,9 @@ So far, we have talked about using Cryptol *value* expressions. However,
 SAWScript can also work with Cryptol *types*. The most direct way to
 refer to a Cryptol type is to use type brackets: `{|` and `|}`. Any
 Cryptol type written between these brackets becomes a `Type` value in
-SAWScript. Some types in Cryptol are *size* types, and isomorphic to
-integers. These can be translated into SAWScript integers with the
-`eval_size` function. For example:
+SAWScript. Some types in Cryptol are *numeric* (also known as *size*)
+types, and isomorphic to integers. These can be translated into
```

Comment:
Cryptol numeric types are not really "isomorphic to integers"; it would be more precise to say "non-negative integers".

If you want to be *really* pedantic, it would be "extended natural numbers" but for clarity it's probably best to elide that detail. Although if we leave out that detail then maybe we should avoid a word with such a precise technical meaning as "isomorphic". So maybe I'd recommend, "Some types in Cryptol are *numeric* (aka *size*) types, and correspond to non-negative integers."

Reply:
Yep, I like your wording suggestion.